examples/graphql-idl-support/graphql_idl_support.bal

```diff
@@ -0,0 +1,14 @@
+import ballerina/io;
+client "./graphql.config.yaml" as foo;
+
+public function main() {
+    do {
```

Comment:
Is it possible to replace `do-on fail` with `check` and/or type checks, so that the BBE will be simpler and more focused on the IDL code?